modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-0-release-notes_{context}"]
+= OADP 1.4.0 release notes
+
+The {oadp-first} 1.4.0 release notes lists resolved issues and known issues.
+
+[id="resolved-issues-1-4-0_{context}"]
+== Resolved issues
+
+.Restore works correctly in OpenShift Container Platform 4.16
+
+Previously, while restoring the deleted application namespace, the restore operation partially failed with the `resource name may not be empty` error in OpenShift Container Platform 4.16.
+With this update, restore works as expected in OpenShift Container Platform 4.16
+link:https://issues.redhat.com/browse/OADP-4075[OADP-4075]
+
+.Data Mover backups works properly in the OpenShift Container Platform 4.16 cluster
+
+Previously, Velero was using the lower version of SDK where the `Spec.SourceVolumeMode` field did not exist. As a consequence, Data Mover backups failed in the OpenShift Container Platform 4.16 cluster on the external snapshotter with v4.2 version. 
+With this update, external snapshotter is upgraded to v7.0 version and later. As a result, backups do not fail in the OpenShift Container Platform 4.16 cluster.
+link:https://issues.redhat.com/browse/OADP-3922[OADP-3922]
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438505[OADP 1.4.0 resolved issues] in Jira.
+
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+
+.Backup fails when checksumAlgorithm is not set for MCG
+
+While performing a backup of any application with Noobaa as the backup location, if the `checksumAlgorithm` configuration parameter is not set, backup fails. To work around this problem, an empty value of `checksumAlgorithm` is added to the Backup Storage Location (BSL) configuration if you have not provided this value.
+The empty value is only added for BSLs that are created using Data Protection Application (DPA) custom resource (CR), and this value is not added if BSLs are created using any other method.
+For more information, see xref:changes-oadp-1-3-to-1-4_oadp-1-4-release-notes[Changes from OADP 1.3 to 1.4].
+link:https://issues.redhat.com/browse/OADP-4274[OADP-4274]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438506[OADP 1.4.0 known issues] in Jira.
+
+
+[id="upgrade-notes-1-4-0_{context}"]
+== Upgrade notes
+
+[NOTE]
+====
+Always upgrade to the next minor version. *Do not* skip versions. To update to a later version, upgrade only one channel at a time. For example, to upgrade from {oadp-first} 1.1 to 1.3, upgrade first to 1.2, and then to 1.3.
+====
+
+[id="changes-oadp-1-3-to-1-4_{context}"]
+=== Changes from OADP 1.3 to 1.4
+
+The Velero server has been updated from version 1.12 to 1.14.
+
+There are no changes required in the following updates:
```

Comment:
please revert change done in this line

Reply:
Please review now

Reply:
![Screenshot from 2024-07-02 16-15-38](https://github.com/openshift/openshift-docs/assets/66965232/837f862a-80c0-4a96-a3ec-70eb64ad708b)

It makes no sense to me. At top level we say there are no changes, but after that, we say there are

I would revert the change to this line and use the template used for other release notes for consistency

Reply:
@mateusoliveira43, I reverted the changes, which I added originally. But to explain there is no change in the DPA, added an admonition for that.
Could you review it and let me know if there are any further inputs? Accordingly, @stillalearner can provide a QE ack and @GroceryBoyJr can complete the peer review after that.
Thanks!

Reply:
![Screenshot from 2024-07-05 15-12-23](https://github.com/openshift/openshift-docs/assets/66965232/0ba7f615-a582-42eb-b28d-a71198bf3fb5)

The note is good, emphasizes to user that no changes are needed in DPA

But is still confusing, as the first bullet says there are changes needed in DPA, which is wrong. This should be fixed (by adding a **no** to the sentence) or removing that sentence

Reply:
oh, now I get it. You were suggesting for the bullet point but it was confusing due to the line number where the suggestion was added.
I have removed "There are changes needed in the Data Protection Application (DPA)." from the 1st bullet point and kept the note emphasizing on there are no changes in the DPA.

Reply:
@mateusoliveira43, could you review it and let me know if there are any further inputs or does it look good?